src/wifi-setup/wifi.js

```diff
@@ -15,6 +15,24 @@ function getStatus() {
   return platform.getWirelessMode();
 }
 
+/**
+ * Get the SSID of the hotspot.
+ *
+ * @returns {string} SSID
+ */
+function getHotspotSsid() {
+  const base = config.get('wifi.ap.ssid_base');
+  const mac = platform.getMacAddress('wlan0');
+  if (!mac) {
+    return base;
+  }
+
+  // Get the last 2 octets of the MAC and create a simple string, e.g. 9E28
```

Comment:
I'd recommend using the last 3 octets, since the first 3 octets are a manufacturer code and the last 3 digits will be unique within a given manufacturer code.

Reply:
I did this based on other wifi networks around me. It seems like most OEMs use a string of 4 digits at the end. Probably easier to remember than 6.